app/models/game.rb

```diff
@@ -7,6 +7,8 @@ class Game < ApplicationRecord
 
   after_create_commit :setup
 
+  scope :top10, -> { select(:total_score, :user_id).distinct.order('total_score DESC').limit(10) }
```

Comment:
los nombres con números en métodos y variables se me hace algo raro puedes ser mas verboso si dejaras el método así 
```suggestion
  scope :top, ->(limit) { select(:total_score, :user_id).distinct.order('total_score DESC').limit(limit) }
```

con eso a la hora de llamar el método seria algo así

```
  @games = Game.top(10)
```